src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -29,6 +29,16 @@ import {OutgoingMessage} from '../../OutgoingMessage.js';
 
 import {SubscriptionManager} from './SubscriptionManager.js';
 
+const EVENT_NAMES = new Set([
```

Comment:
What if we moved `EVENT_NAMES` to https://github.com/GoogleChromeLabs/chromium-bidi/blob/793c0af2def70a2fce67d0f79ae60b197bcc1fb8/src/protocol/chromium-bidi.ts#L21, alongside the `EventNames` type?

I think it would be easier to maintain / keep in sync (especially when/as new modules are added).

Reply:
I just moved it to the top in this PR, will think about if there is a way to make it seems less.